package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@retailmenot/anchor",
-  "version": "1.3.7",
+  "version": "1.4.0",
```

Comment:
just curious why jump a new minor version and a couple of bug fix versions?

Reply:
This introduces breaking changes with the Typog..y component, but it's merely a consistency fix with a lower level dependency so I felt like a good middle-ground would be a minor release.